### DISCUSSION TITLE
Locomotive Scroll

### DISCUSSION THREAD

**Discussion by cyprianwaclaw** [nuxt/nuxt#17551] [Questions]
How use locomotive scroll in nuxt 3 project ?

**Comment by alanfrancis442** [comment]
check this [link](https://dev.to/didomarchet/how-to-locomotive-scroll-and-nuxt-with-example-and-gotchas-1gfi)

**Comment by arashsheyda** [comment]
built a WIP wrapper around it: [nuxt-locomotive-scroll](https://www.npmjs.com/package/nuxt-locomotive-scroll)
